sgn.json

```diff
@@ -58,5 +58,44 @@
             "href": "http://mathworld.wolfram.com/Sign.html",
             "title": "Sign explained by Wolfram MathWorld"
         }
-    ]
+    ],
+    "process_graph": {
+        "gt0": {
+            "x": {
+                "from_parameter": "x"
+            },
+            "y": 0
+        },
+        "lt0": {
+            "x": {
+                "from_parameter": "x"
+            },
+            "y": 0
+        },
+        "if_gt0": {
+            "process_id": "if",
+            "arguments": {
+                "value": {
+                    "from_node": "gt0"
+                },
+                "accept": 1,
+                "reject": {
```

Comment:
why not just `"reject" : 0`  ? With `"from_parameter": "x"` it looks bit confusing

Reply:
It can be either 0 or null and that's what is in "from_parameter": "x" after the lt/gt checks.